src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.h

```diff
@@ -177,16 +180,14 @@ class GenericThreadStackManagerImpl_OpenThread
 #if CHIP_DEVICE_CONFIG_ENABLE_THREAD_DNS_CLIENT
 #if CHIP_DEVICE_CONFIG_ENABLE_THREAD_COMMISSIONABLE_DISCOVERY
     // Thread supports both operational and commissionable discovery, so buffers sizes must be worst case.
-    static constexpr uint8_t kMaxDnsServiceTxtEntriesNumber =
-        chip::Mdns::CommissionAdvertisingParameters::kNumAdvertisingTxtEntries;
-    static constexpr uint8_t kMaxDnsServiceTxtValueSize = chip::Mdns::CommissionAdvertisingParameters::kTxtMaxValueSize;
-    static constexpr uint8_t kMaxDnsServiceTxtKeySize   = chip::Mdns::OperationalAdvertisingParameters::kTxtMaxKeySize;
+    static constexpr uint8_t kMaxDnsServiceTxtEntriesNumber = Mdns::CommissionAdvertisingParameters::kTxtMaxNumber;
+    static constexpr uint8_t kMaxDnsServiceTxtValueSize     = Mdns::CommissionAdvertisingParameters::kTxtMaxValueSize;
+    static constexpr uint8_t kMaxDnsServiceTxtKeySize       = Mdns::OperationalAdvertisingParameters::kTxtMaxKeySize;
```

Comment:
Again, this should be using std::max instead of hardcoding which fields are bigger for commissioning vs operational advertisement.

Reply:
Here, the value for operational adv. is purposefully used instead of the one for commissionable adv. (although the latter is greater) because it would allocate too much memory on stack otherwise. This will be fixed in a later PR.

Reply:
Please leave a TODO comment?

Reply:
Sure!